src/include/txn.i

```diff
@@ -806,8 +806,9 @@ __wt_txn_begin(WT_SESSION_IMPL *session, const char *cfg[])
     txn->isolation = session->isolation;
     txn->txn_logsync = S2C(session)->txn_logsync;
 
-    if (cfg != NULL)
-        WT_RET(__wt_txn_config(session, cfg));
+    WT_ASSERT(session, !F_ISSET(txn, WT_TXN_RUNNING));
```

Comment:
You brought in an assert here that wasn't part of the original change.

Reply:
Thanks. Will create a new PR.